lib/git_worktree.rb

```diff
@@ -258,6 +258,22 @@ def checkout
     end
   end
 
+  # Ping the git repository endpoint to verify that the network connection is still up.
+  #
```

Comment:
We might want a comment here about trying to use the built-in check_connection but getting segfaults and linking to the upstream issue.

Reply:
Done.